crates/apps/src/lib/cli/context.rs

```diff
@@ -621,8 +621,5 @@ impl ArgFromMutContext for BalanceOwner {
                 ExtendedViewingKey::arg_from_mut_ctx(ctx, raw)
                     .map(Self::FullViewingKey)
             })
-            .or_else(|_| {
```

Comment:
why was this removed?

Reply:
I believe it was only used for the pinned balance